### PR TITLE
Add ignore for ServerKeepAlivePacket

### PR DESCRIPTION
**Pull request by rtm516**
This adds ServerKeepAlivePacket as a packet to ignore as its likely not to be implemented soon and this will prevent it from spamming console with `Could not find packet for ServerKeepAlivePacket`

### CODE REVIEW

File: connector/src/main/java/org/geysermc/connector/network/translators/PacketTranslatorRegistry.java [hunk 1]
```diff
@@ -73,6 +73,7 @@
         }
 
         IGNORED_PACKETS.add(ServerUpdateLightPacket.class);
+        IGNORED_PACKETS.add(ServerKeepAlivePacket.class);
```
Comment (DoctorMacc) [inline connector/src/main/java/org/geysermc/connector/network/translators/PacketTranslatorRegistry.java:76]: Probably alphabetize this list?

**Comment by DoctorMacc** [comment]
`The server will frequently send out a keep-alive, each containing a random ID. The client must respond with the same packet.` How does Geyser manage this keep alive currently?

**Comment by RednedEpic** [comment]
> `The server will frequently send out a keep-alive, each containing a random ID. The client must respond with the same packet.` How does Geyser manage this keep alive currently?

It doesn't currently. I think we should instead handle this packet instead of ignoring it.

**Comment by Tim203** [comment]
If you ignore the packet for more then 30 seconds, you will time out. So it is already handled.

**Comment by DoctorMacc** [comment]
https://github.com/Steveice10/MCProtocolLib/blob/master/src/main/java/com/github/steveice10/mc/protocol/ClientListener.java is the link Tim sent me. Once the packet ignore list is alphabetized I'll approve this.

**Pull request merged**